# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/gl/GLLibraryEGL.h
# Commit: 16c5dd2cfd60
# Full Hash: 16c5dd2cfd600a62c80ca17cfb35c0868e016056
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-15 21:48:38
# Regressor Bug: 1632249
# File Overlap Count: 1
# Description:
#   Bug 1632249 - Support compositing out-of-process WebGL. r=handyman,lsalzman,nical,geckoview-reviewers,agi,imanol
#   
#   * Majorly simplity CanvasRenderer
#   * Replace GLScreenBuffer with trivial GLSwapChain
#   * Use descriptor structs so that future SharedSurface changes aren't so painful
# ==============================================================================

diff -r ca8139d58d58 -r 16c5dd2cfd60 gfx/gl/GLLibraryEGL.h
--- a/gfx/gl/GLLibraryEGL.h	Fri Jun 12 21:53:14 2020 +0000
+++ b/gfx/gl/GLLibraryEGL.h	Mon Jun 15 18:25:55 2020 +0000
@@ -385,21 +385,21 @@
     return mIsWARP;
   }
 
-  bool HasKHRImageBase() {
+  bool HasKHRImageBase() const {
     return IsExtensionSupported(KHR_image) ||
            IsExtensionSupported(KHR_image_base);
   }
 
-  bool HasKHRImagePixmap() {
+  bool HasKHRImagePixmap() const {
     return IsExtensionSupported(KHR_image) ||
            IsExtensionSupported(KHR_image_pixmap);
   }
 
-  bool HasKHRImageTexture2D() {
+  bool HasKHRImageTexture2D() const {
     return IsExtensionSupported(KHR_gl_texture_2D_image);
   }
 
-  bool HasANGLESurfaceD3DTexture2DShareHandle() {
+  bool HasANGLESurfaceD3DTexture2DShareHandle() const {
     return IsExtensionSupported(ANGLE_surface_d3d_texture_2d_share_handle);
   }
 